client/app/components/collection-item/component.js

```diff
@@ -2,10 +2,13 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
     tagName: null,
-    classNameBindings: ['selected:item-row-selected'],
+    classNameBindings: ['rowSelected:item-row-selected'],
     cardView: true,
     item: null,
     selected: false,
+    rowSelected: Ember.computed('organizeMode', 'selected', function(){
+        return !this.get('organizeMode') ? false : this.get('selected');
```

Comment:
Also can you add `organizeMode` as a property in the component? It is better to have all of a component's properties explicitly defined.